examples/fusionauth/architect.yml

```diff
@@ -0,0 +1,81 @@
+name: architect/fusionauth
+author: Architect.io
+
+interfaces:
+  http: ${ services.fusionapi.interfaces.http.url }
+  mgmt: ${ services.fusionapi.interfaces.mgmt.url }
+
+parameters:
+  DATABASE_ROOT_USER:
+    required: false
+    description: The database root user that is used to create the FusionAuth schema and FusionAuth user.
+  DATABASE_ROOT_PASSWORD:
+    required: false
+    description: The database root password that is used to create the FusionAuth schema and FusionAuth user. It is recommended to leave the value of this variable empty.
+  DATABASE_USER:
+    default: fusionauth
+    description: The database user that will be created during configuration to own the FusionAuth schema and to connect to the database at FusionAuth runtime.
+  DATABASE_PASSWORD:
+    default: fusionauth
+    description: The database password that will be created during configuration to own the FusionAuth schema and to connect to the database at FusionAuth runtime.
+  FUSIONAUTH_ADDITIONAL_JAVA_ARGS:
+    required: false
+    description: Any additional arguments that you want to pass to the Java VM where this service will run.
+  FUSIONAUTH_COOKIE_SAME_SITE_POLICY:
+    required: false
+    description: The value to use in the Same-Site cookie attribute for cookies set by FusionAuth. None, Lax, Strict
+  FUSIONAUTH_MEMORY:
+    default: 512M
+    description: The amount of memory to allocate to Java VM where this service will run. Use the M or G suffix to denote megabytes and gigabytes respectively.
+  FUSIONAUTH_RUNTIME_MODE:
+    default: development
+    description: The FusionAuth runtime mode. The runtime mode causes some features to be modified based upon this configuration. development, production
+  FUSIONAUTH_SEARCH_ENGINE_TYPE:
+    default: database
+    description: The search engine type. When configuring the search engine type to elasticsearch, you must configure the fusionauth-app.search-servers property as well. database, elasticsearch
+
+services:
+
+  db:
+    description: Fusion Database
+    language: postgresql
+    image: postgres:9.6
+    environment:
+      POSTGRES_USER: ${ parameters.DATABASE_USER }
+      POSTGRES_PASSWORD: ${ parameters.DATABASE_PASSWORD }
+      POSTGRES_DB: fusionauth
+    interfaces:
+      main:
+        port: 5432
+        protocol: postgresql
+
+  fusionapi:
+    description: Fusion API
+    language: go
```

Comment:
Wasn't this a Java service?

Reply:
Haha! Indeed it was. I thought it was go early on because Matt Gowie's client was in go but clearly forgot to change it when I realized it was java. Thanks